src/gui/main-styles.js

```diff
@@ -345,6 +345,20 @@ styles.registerStyle('main', () => {
 			'-webkit-overflow-scrolling': 'touch',
 			'-ms-overflow-style': '-ms-autohiding-scrollbar'
 		},
+		'.scroll-webkit-scrollbar': {
```

Comment:
Don't we have other scrollbar classes already? why do we need these ones? What do they do? Comment would be lovely